tests/test.rs

```diff
@@ -295,6 +295,15 @@ fn test_trans_with_nested_trans() {
 }
 
 #[test]
+#[should_panic(expected = "active transaction")]
+fn test_trans_with_savepoints() {
+    let conn = or_panic!(Connection::connect("postgres://postgres@localhost", SslMode::None));
+    let trans = or_panic!(conn.transaction());
+    let _sp = or_panic!(trans.savepoint("custom"));
```

Comment:
Can you add `commit()` calls to the savepoints here to make sure that they commit successfully?